doc/source/whatsnew/v0.17.0.txt

```diff
@@ -385,6 +385,8 @@ Bug Fixes
 
 - Bug in operator equal on Index not being consistent with Series (:issue:`9947`)
```

Comment:
I realized that you have this listed twice in the whatsnew in Bug Fixes and in the separate section. Pls remove from bug fixes (and just have it in the separate section).